Docs/neurons.md

```diff
@@ -273,6 +273,62 @@ Here is another example brain whit use the `neurotimer` neuron. In this scenario
           - "You asked me to remind you to {{ kalliope_memory['remember'] }} {{ kalliope_memory['time'] }} minutes ago"
 ```
 
+### Get the last order generated TTS message
+
+Kalliope will save in memory automatically:
+- the last generated TTS message . 
+- the last caught order
+
+To get the last generated message, use the key `{{ kalliope_memory['kalliope_last_tts_message'] }}` in your synapse.
```

Comment:
Here we define 2 news restricted keys for the kalliope_memory.
What happens if an user use those keywords in a neuron : 
```
...
kalliope_memory:
          key_name_in_memory: "{{ output_variable_from_neuron }}"
```
?

Reply:
If the user call those keys from a neuron, he will get the last tts message. Not a big deal. Just useless. 
In many case the user will set a hook that make kalliope say something before catching the order. This will be the last message so the user will get "yes, sir?"

Reply:
Yes if the user calls it, but if he defines it like (my previous exemple was not relevant) : 
```
...
kalliope_memory:
          kalliope_last_tts_message: "{{ output_variable_from_neuron }}"
```

Reply:
He will not override it. It's saved at the end of the neuron execution process